### PR TITLE
Sim2dist

### PR DESCRIPTION
**Pull request by AWBroscius**
Implementation in python of the conversion methods for similarity coefficients from [sim2diss](https://rdrr.io/cran/smacof/man/sim2diss.html)

### CODE REVIEW

**Review by FarnazH — CHANGES_REQUESTED** [review]
This functionality should be implemented in the `converter.py` module.

**Review by FanwangM — CHANGES_REQUESTED** [review]
This is a big improvement and I enjoy reading the codes a lot. Nice job!

Sorry for being so picky, but we all want to make the package nice and easy to read/maintain.

File: DiverseSelector/converter.py [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+
+"""Utils module."""
+
+import numpy as np
```
[the remark this answers is not in the document]
Comment (AWBroscius) [reply DiverseSelector/converter.py:26]: I am not sure what you want changed here. Importing `numpy` automatically imports `ndarray`.

**Comment by AWBroscius** [comment]
Hi @FanwangM I just pushed another commit that addresses most of your comments. Thank you for the advice. Feel free to look it over again when you have the chance.

File: DiverseSelector/converter.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (FarnazH) [reply DiverseSelector/converter.py:27]: The `np.ndarray` is used, so there is no need to import `ndarray`.

**Comment by FanwangM** [comment]
I am good with current PR. Do you have any other comments for this PR?  @FarnazH

**Review by FarnazH — APPROVED** [review]
Thanks, @AWBroscius for your contribution. The code looks good to me overall, but I will most likely add some changes later to make it more consistent with the rest of the code.

**Pull request merged**